eng/tools/specs-model/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { exit } from "node:process";
+import { getSpecsModel } from "./getSpecsModel.js";
+
+function getUsage(): string {
+  return (
+    "  Usage: npx get-specs-model <path-to-specs-file-or-directory>\n" +
+    "Returns: JSON metadata for the input file or directory.\n" +
+    "\n" +
+    "The input path:\n" +
+    "- Must be an absolute or relative path to local clone of the https://github.com/Azure/azure-rest-api-specs or https://github.com/Azure/azure-rest-api-specs-pr repo.\n" +
+    "- Must point to the <local_clone>/specification directory or one of its descendants.\n" +
+    "\n" +
+    "Example: npx get-specs-model $HOME/repos/azure-rest-api-specs/specification\n" +
+    "Returns: JSON with metadata for the entire 'specification' directory of the local clone of 'azure-rest-api-specs' repo.\n"
+  );
+}
+
+export async function main() {
+  const arguments_: string[] = process.argv.slice(2);
```

Comment:
Why the trailing underscore on `arguments_`?  Is this a coding convention to signify something?

Reply:
unicorn linter forbade using `args` (no abbreviations) and arguments appears to be reserved keyword. I am happy to disable this rule for `args` and use `args`.

Reply:
Fixed with linter config update:

```
      // We allow some abbreviations that we like.
      // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prevent-abbreviations.md
      "unicorn/prevent-abbreviations": {
        allowList: {
          args: true,
        },
      },
```

Reply:
I agree in principle, but I'm not sure how "prevent abbreviations" will work in practice.  I think some abbreviations are good (e.g. `args`, `dir`, etc), while others are bad (e.g. `cust` instead of `customer`).  It's more a matter of taste than "anything not in the dictionary should be banned".

But I'm willing to give it a try, so I'm fine with disabling the rule for `args` for now.

Reply:
Yeah, I have similar attitude - if it turns out this rule is too against our preferences, we can outright disable it. But let's give it a try.